tests/simulator/user_modeling/test_preference_model.py

```diff
@@ -40,68 +37,52 @@ def preference_model_sip() -> PreferenceModel:
         domain,
         item_collection,
         ratings,
-        PreferenceModelVariant.SIP,
         historical_user_id="13",
     )
 
 
-# Peronal Knowledge Graph preference model variant.
-@pytest.fixture
-def preference_model_pkg() -> PreferenceModel:
-    domain = Domain(DOMAIN_YAML_FILE)
-    item_collection = ItemCollection()
-    item_collection.load_items_csv(
-        ITEMS_CSV_FILE,
-        id_col="movieId",
-        domain=domain,
-        domain_mapping=DOMAIN_MAPPING,
-    )
-    ratings = Ratings()
-    ratings.load_ratings_csv(RATINGS_CSV_FILE)
-    return PreferenceModel(
-        domain,
-        item_collection,
-        ratings,
-        PreferenceModelVariant.PKG,
-        historical_user_id="13",
-    )
+@pytest.mark.parametrize("item_id, expected", [("377", True), ("591", False)])
+def test_is_item_consumed(
+    item_id: str, expected: bool, preference_model: PreferenceModel
+) -> None:
+    assert expected == preference_model.is_item_consumed(item_id)
 
 
-def test_initial_item_preferences_sip(preference_model_sip) -> None:
-    # TODO this is the value in the original rating file, but it'll be a bit
-    # tricky to test when only a sample of the original ratings is used.
-    assert preference_model_sip.get_item_preference("356") == 1.0
+def test_get_item_preference(preference_model: PreferenceModel) -> None:
+    preference = preference_model.get_item_preference("527")
+    assert preference == 1.0 or preference == -1.0
 
 
-def test_get_slotvalue_preference_sip(preference_model_sip) -> None:
-    preference = preference_model_sip.get_slotvalue_preference(
-        "GENRE", "Comedy"
-    )
-    assert -1 <= preference <= 1
-
-
-def test_get_slotvalue_preference_pkg(preference_model_pkg) -> None:
-    preference = preference_model_pkg.get_slotvalue_preference("GENRE", "Drama")
-    assert preference == 0.6
+def test_get_item_preference_error(preference_model: PreferenceModel) -> None:
```

Comment:
Should we indicate in the method name that we're testing preference for a non-existing item? e.g., `test_get_item_preference_nonexisting_item`